docs/source/en/big_models.md

```diff
@@ -14,110 +14,205 @@ rendered properly in your Markdown viewer.
 
 -->
 
-# Instantiating a big model
+# Instantiate a big model
 
-When you want to use a very big pretrained model, one challenge is to minimize the use of the RAM. The usual workflow
-from PyTorch is:
+A barrier to accessing very large pretrained models is the amount of memory the require. When you load a pretrained PyTorch model, you usually:
 
-1. Create your model with random weights.
+1. Create a model with random weights.
 2. Load your pretrained weights.
 3. Put those pretrained weights in your random model.
 
-Step 1 and 2 both require a full version of the model in memory, which is not a problem in most cases, but if your model starts weighing several GigaBytes, those two copies can make you get out of RAM. Even worse, if you are using `torch.distributed` to launch a distributed training, each process will load the pretrained model and store these two copies in RAM.
+The first two steps both require a full version of the model in memory and if your model starts weighing several GBs, you may not have enough memory for two copies of the model. This problem is amplified in distributed training environments because each process loads the pretrained model and stores two copies in memory.
```

Comment:
```suggestion
The first two steps both require a full version of the model in memory and if your model weights several GBs, you may not have enough memory for two copies of the model. This problem is amplified in distributed training environments because each process loads the pretrained model and stores two copies in memory.
```